src/main/java/org/opentripplanner/street/model/edge/VehicleParkingEdge.java

```diff
@@ -137,20 +155,29 @@ private State[] traversePark(State s0) {
         return State.empty();
       }
 
-      return traversePark(s0, preferences.bike().parkCost(), preferences.bike().parkTime());
+      return traversePark(
+        s0,
+        preferences.bike().parking().parkCost().toSeconds(),
+        (int) preferences.bike().parking().parkTime().toSeconds()
+      );
     } else if (streetMode.includesDriving()) {
-      return traversePark(s0, preferences.car().parkCost(), preferences.car().parkTime());
+      return traversePark(
+        s0,
+        preferences.car().parking().parkCost().toSeconds(),
+        (int) preferences.car().parking().parkTime().toSeconds()
+      );
```

Comment:
Can you pass in Cost and Duration so you only convert to seconds once?

Reply:
Done